test/test_MVU.sh

```diff
@@ -316,7 +311,8 @@ add_exclude 9.6 10 test/sql/partitions.sql
 #(cd $(dirname $0)/..; pg_prove -v --pset tuples_only=1 test/sql/throwtap.sql)
 
 export EXCLUDE_TEST_FILES
-run_make clean test
+run_make clean
+make test
```

Comment:
Shouldn't we consistently use `run_make`?

Reply:
Perhaps, but I got failures that the user named `root` did not exist. Like `PGUSER` wasn't propagating. This fixed that issue.

Reply:
Hrm... now I'm wondering if we need run_make at all. I don't remember why I created it to begin with. Either way, it'd be best to be consistent on how we're running make.

Reply:
Removed in 4479938 and working in 84d35e7.